Description: Please install ALSA plugin in the multiarch directory
Author: Steve Langasek <steve.langasek@canonical.com>
Origin: Steve Langasek <steve.langasek@canonical.com>
Updated by Nobuhiro Iwamatsu <iwamatsu@debian.org>
Debian BTS: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=634976

--- a/Makefile.am	2011-12-26 19:08:38.000000000 +0900
+++ b/Makefile.am	2011-12-26 19:13:45.000000000 +0900
@@ -347,7 +347,7 @@
 			proximity/proximity.conf
 
 if ALSA
-alsadir = $(libdir)/alsa-lib
+alsadir = $(ALSA_LIBDIR)/alsa-lib
 
 alsa_LTLIBRARIES = audio/libasound_module_pcm_bluetooth.la \
 				audio/libasound_module_ctl_bluetooth.la
--- a/acinclude.m4
+++ b/acinclude.m4
@@ -135,6 +135,8 @@ AC_DEFUN([AC_PATH_ALSA], [
 	AC_CHECK_LIB(rt, clock_gettime, ALSA_LIBS="$ALSA_LIBS -lrt", alsa_found=no)
 	AC_SUBST(ALSA_CFLAGS)
 	AC_SUBST(ALSA_LIBS)
+	ALSA_LIBDIR=`$PKG_CONFIG --variable=libdir alsa 2>/dev/null`
+	AC_SUBST(ALSA_LIBDIR)
 ])
 
 AC_DEFUN([AC_PATH_USB], [
